layout-management-services/src/main/java/org/exoplatform/layoutmanagement/rest/SiteNavigationRestService.java

```diff
@@ -203,7 +203,8 @@ public Response createNode(@Parameter(description = "navigation node id")
                                     Visibility.TEMPORAL,
                                     StringUtils.isBlank(pageRef) ? null : PageKey.parse(pageRef),
                                     null,
-                                    target);
+                                    target,
```

Comment:
should we also update the last modified date for these operation:

- move node
- update related page (permissions, link ..)

to be checked with @mzorai